src/net_processing.h

```diff
@@ -66,9 +66,9 @@ struct Peer {
     /** List of block ids we still have announce.
      * There is no final sorting before sending, as they are always sent
      * immediately and in the order requested. */
-    std::vector<uint256> vInventoryBlockToSend GUARDED_BY(m_block_inv_mutex);
+    std::vector<uint256> m_blocks_for_inv_relay GUARDED_BY(m_block_inv_mutex);
     /** List of headers to relay */
-    std::vector<uint256> vBlockHashesToAnnounce GUARDED_BY(m_block_inv_mutex);
+    std::vector<uint256> m_blocks_for_headers_relay GUARDED_BY(m_block_inv_mutex);
```

Comment:
22c2b3eff601cbc1aed2e466502bdc5549e11ae2

I don't think this name is correct. Headers relay isn't implied. It might very well fall back to inv relay. Most importantly when the peer asked us to. Though, there are other reasons, too.

Reply:
We hope to announce these blocks via headers. If we can't, we'll move the hashes onto the `m_blocks_for_inv_relay` and then announce via inv.

I've updated the comment. Let me know if you think it's clear enough now.

Reply:
Thanks. Looks good now.